crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -961,10 +962,30 @@ def upgrade_cluster(founder, cluster)
         Rails.logger.info("Nodes in cluster #{cluster} successfully upgraded")
       end
 
+      # Rabbitmq uses native clustering in SOC8, we need to adapt the configuration
+      def upgrade_rabbitmq_setup
+        rabbit_cluster_members = ::Node.find(
+          "run_list_map:pacemaker-cluster-member AND run_list_map:rabbitmq-server "
+        )
+        return if rabbit_cluster_members.empty?
+
+        prop = Proposal.find_by(barclamp: "rabbitmq", name: "default")
+        unless prop.raw_data["attributes"]["rabbitmq"]["cluster"]
+          prop.raw_data["attributes"]["rabbitmq"]["cluster"] = true
```

Comment:
You also want to save the `["rabbitmq"]["erlang_cookie"]` here so its in sync with the node one.

Reply:
Do you think I should pass this one to the nodes?